tests/quantization/aqlm_integration/test_aqlm.py

```diff
@@ -178,6 +182,56 @@ def test_quantized_model_multi_gpu(self):
 
         self.assertTrue(set(quantized_model.hf_device_map.values()) == {0, 1})
 
-        output = quantized_model.generate(**input_ids, max_new_tokens=40)
+        output = quantized_model.generate(**input_ids, max_new_tokens=self.max_new_tokens)
 
         self.assertEqual(self.tokenizer.decode(output[0], skip_special_tokens=True), self.EXPECTED_OUTPUT)
+
+    @unittest.skipUnless(
+        is_aqlm_available() and version.parse(importlib.metadata.version("aqlm")) >= version.parse("1.0.3"),
+        "test requires `aqlm>=1.0.3`",
+    )
+    def test_quantized_model_compile(self):
```

Comment:
Loving this test ❤️

Reply:
@ArthurZucker @BlackSamorez  The problem with it that it fails :) . See [this](https://github.com/huggingface/transformers/pull/29079#issuecomment-1959565957). So, advice needed on what to do here

Reply:
I don't think it is super important that the outputs match for quantized models no? Distributions are the same, but kernels / ops are not run in the same order. It's small but could explain this? 
Would just add a long generation and make sure it still makes sense!

Reply:
I don't really know how to automatically check if text makes sense.
Alternatively, I've shortened the generation length from 40 tokens to 32 and it matches perfectly on `RTX 3090`, `RTX 2080ti` and `a6000`. Maybe we could just leave it as is since the tests above are exact match anyway.

Reply:
fine with me 😉